src/components/app.tsx

```diff
@@ -13,7 +13,7 @@ import { State } from '../constants/default-state';
 import './app.css';
 import Header from './header';
 import InputPanel from './input-panel';
-import Sidebar from './sidebar';
+import Sidebar from './Sidebar';
```

Comment:
we generally don't capitalize file names

Reply:
Okay!